### PR TITLE
Runners name may not be unique in a workflow

### LINKED ISSUE #17589: Indicate if runner names are temporaly unique within a given scope
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-object-50
https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

### What part(s) of the article would you like to see updated?

https://github.com/github/docs/blob/e93864362002a180c7ec17eefdb8cb49efea9e40/data/reusables/actions/runner-name-description.md?plain=1#L1

It'd be nice if something clearly indicated if a runner name is temporaly unique within a given scope.

i.e. Can two runners running jobs at the same time for the same repository have the same name?

I think the answer is no. But I'm not sure.

### Additional information

https://github.community/t/ref-head-in-reusable-workflows/203690/95?u=jsoref

### PR DESCRIPTION
**Pull request by electron97**
- Currently the docs doesn't specify if the runners name can be unique or not in a workflow.
- Made changes accordingly to add a note that runners name may not be unique in a workflow as runners at repository and organization level could use the same name.

Closes: #17589 

### Screenshots:

**Before Fix**
Screenshot 2023-08-26 at 1 34 52 PM

**After Fix**
Screenshot 2023-08-26 at 1 34 17 PM

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

File: data/reusables/actions/runner-name-description.md [hunk 1]
```diff
@@ -1 +1 @@
-The name of the runner executing the job.
+The name of the runner executing the job. Also the name may not be unique in a workflow run as runners at repository and organization level could use the same name.
```
Comment (jsoref) [inline data/reusables/actions/runner-name-description.md:1]: Technically, I believe it also could be an `enterprise level` runner. I don't know if that requires conditional markup or not.

Comment (jsoref) [inline data/reusables/actions/runner-name-description.md:1]: I'd probably suggest not using `Also`. My default preference would be `Note that`, but I'd defer to the docs owners.

Comment (0981241725) [reply data/reusables/actions/runner-name-description.md:1]: []()

**Review by sundumanikanta — COMMENTED** [review]
https://github.com/github/haikus-for-codespaces/pull/214#issue-1658643432

File: data/reusables/actions/runner-name-description.md (hunk 1 above)
Comment (hubwriter) [inline data/reusables/actions/runner-name-description.md:1]: I'd suggest this tiny change:

```suggestion
The name of the runner executing the job. This name may not be unique in a workflow run as runners at repository and organization level could use the same name.
```

**Review by hubwriter — DISMISSED** [review]
@electron97 - Many thanks for raising this PR. 👍 

I've made a tiny change. Good to get this added. Thanks for your help! I'll get this merged.

🚀

File: data/reusables/actions/runner-name-description.md [hunk 2]
```diff
@@ -1 +1 @@
-The name of the runner executing the job.
+The name of the runner executing the job. This name may not be unique in a workflow run as runners at repository and organization level could use the same name.
```
Comment (SiaraMist) [inline data/reusables/actions/runner-name-description.md:1]:
```suggestion
The name of the runner executing the job. This name may not be unique in a workflow run as runners at the repository and organization levels could use the same name.
```

**Review by SiaraMist — APPROVED** [review]

**Pull request merged**